### PR TITLE
[GL-1694] Adding scientific tests for Imputation Pipeline

### PR DESCRIPTION
**Pull request by cheyenne-gold**
Successful scientific test run: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/14312/

### CODE REVIEW

**Comment by cheyenne-gold** [comment]
retest this please

File: pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Imputation.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation/scientific/vcfs/merged_input_samples.vcf.gz",
+  "Imputation.multi_sample_vcf_index": "gs://broad-gotc-test-storage/imputation/scientific/vcfs/merged_input_samples.vcf.gz.tbi",
+  "Imputation.ref_dict": "gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.dict",
+  "Imputation.reference_panel_path": "gs://broad-gotc-test-storage/imputation/1000G_reference_panel/",
+  "Imputation.contigs": [ "21", "22" ],
```
Comment (jessicaway) [inline pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json:6]: @gbggrant do we want to run on more chromosomes for this ?

File: pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "Imputation.genetic_maps_eagle": "gs://broad-gotc-test-storage/imputation/eagle_genetic_map/genetic_map_hg19_withX.txt.gz",
```
Comment (jessicaway) [inline pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json:7]:
```suggestion
  "Imputation.genetic_maps_eagle": "gs://broad-references/hg19/v0/eagle_genetic_map_hg19_withX.txt.gz",
```

File: pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "Imputation.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation/scientific/vcfs/merged_input_samples.vcf.gz",
+  "Imputation.multi_sample_vcf_index": "gs://broad-gotc-test-storage/imputation/scientific/vcfs/merged_input_samples.vcf.gz.tbi",
+  "Imputation.ref_dict": "gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.dict",
+  "Imputation.reference_panel_path": "gs://broad-gotc-test-storage/imputation/1000G_reference_panel/",
```
Comment (jessicaway) [inline pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json:5]:
```suggestion
  "Imputation.reference_panel_path": "gs://broad-references/hg19/v0/1000G_reference_panel/",
```

**Review by jessicaway — APPROVED** [review]
minor comments, but looks good!

File: pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "Imputation.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation/scientific/vcfs/merged_input_samples.vcf.gz",
```
[the remark this answers is not in the document]
Comment (cheyenne-gold) [reply pipelines/broad/arrays/imputation/test_inputs/Scientific/multi_sample_test_data_NOT_PUBLIC.json:2]: [This thread ](https://broadinstitute.slack.com/archives/CESEYJW9W/p1633042424116800) is what lead us to test multi-sample for the scientific test. I will leave this multi-sample test the way it is and add scientific another test for the single samples with all chromosomes being tested.

**Pull request merged**